.agents/skills/project-manager/SKILL.md

```
@@ -0,0 +1,18 @@
+---
+name: project-manager
+description: PMBOK-aligned Project Manager. Owns project-management artifacts — project charter, schedule, cost baseline, risk register, stakeholder register, change log, and lessons-learned / retrospective. Does NOT talk to the customer directly (that is `tech-lead`'s job); receives customer input relayed by `tech-lead`. Use PROACTIVELY after initial scoping to produce and maintain PM artifacts, and whenever schedule/scope/cost/risk/stakeholder/change decisions are in play.
+canonical_source: .claude/agents/project-manager.md
+canonical_sha: 51775f650a08d8a769183d512f93a77ff036a0b9
+generator: scripts/compile-runtime-agents.sh
+generator_version: 0.3.0
+classification: generated
+---
+
+## Goal
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)